dockstore-webservice/src/main/java/io/dockstore/webservice/resources/CollectionResource.java

```diff
@@ -78,18 +78,18 @@ public CollectionResource(SessionFactory sessionFactory) {
     /**
      * TODO: Path looks a bit weird
      */
-    @PUT
+    @POST
     @Timed
     @UnitOfWork
     @Override
     @Path("/collections/{collectionId}/aliases")
     @ApiOperation(nickname = "updateCollectionAliases", value = "Update the aliases linked to a Collection in Dockstore.", authorizations = {
```

Comment:
swagger documentation is now out-of-date (i.e. if switching from `update` to `add` as a verb, should update swagger docs too here and in the ApiParam)

Reply:
fixed